app/tests/test_employee.py

```diff
@@ -430,11 +430,16 @@ def test_standup_post(self):
             "name": "Standup",
             "team": self.team1.id,
             "organization": self.organization.id,
-            "created_at": "2023-01-12T01:52:00+05:00",
+            "time": "01:52:00+05:00",
         }
         response = self.client.post("/api/standup/", data=standup_data)
         self.assertEqual(response.status_code, status.HTTP_201_CREATED)
 
+    def test_standup_get_members(self):
+        self.client.force_login(self.owner)
+        response = self.client.get(f"/api/standup/{self.standup.id}/members/")
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
```

Comment:
it is good idea to not only validate status_code but also validate response data against your test data

Reply:
@AizaTariq97 are we handling negative tests?